executor/scripts/build_local.py

```
@@ -125,19 +125,152 @@ def restore_version_source(original_content: str) -> None:
+        sdk_version = getattr(claude_agent_sdk, "__version__", None)
+    except ImportError:
+        sdk_version = None
+
+    print(
+        f"Looking for Windows Claude CLI binary (sdk version: {sdk_version or 'latest'})..."
+    )
+
+    try:
+        # Query PyPI API for package info
+        pypi_url = "https://pypi.org/pypi/claude-agent-sdk/json"
+        with urllib.request.urlopen(pypi_url, timeout=30) as response:
```

Ruff (0.14.14)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)